### PR TITLE
Texture example is added.

### PR DESCRIPTION
**Pull request by gungorbasa**

### CODE REVIEW

**Comment by onmyway133** [comment]
@gungorbasa Hi, thanks for the PR. I'm a bit busy now but I will try to look into this weekend

File: Sources/iOS/IndexPathConverter.swift [hunk 1]
```diff
@@ -1,35 +1,39 @@
...
-    return ChangeWithIndexPath(
-      inserts: inserts,
-      deletes: deletes,
-      replaces: replaces,
-      moves: moves
-    )
-  }
+    }
+    
+    public func convert<T>(changes: [Change<T>], section: Int) -> ChangeWithIndexPath {
```
Comment (onmyway133) [inline Sources/iOS/IndexPathConverter.swift]: I have convention of `2 spaces indentation`, can you use that?

File: Sources/iOS/IndexPathConverter.swift [hunk 2]
```diff
@@ -1,35 +1,39 @@
...
+            inserts: inserts,
+            deletes: deletes,
+            replaces: replaces,
+            moves: moves
+        )
+    }
 }
 
 extension Int {
-  fileprivate func toIndexPath(section: Int) -> IndexPath {
```
Comment (onmyway133) [inline Sources/iOS/IndexPathConverter.swift]: Same here, we need to follow project convention :)

File: Sources/iOS/IndexPathConverter.swift (hunk 1 above)
Comment (gungorbasa) [reply Sources/iOS/IndexPathConverter.swift]: Sure, sorry for my sloppiness. I will fix it.

**Comment by onmyway133** [comment]
@gungorbasa This is great 🚀

**Comment by onmyway133** [comment]
@gungorbasa `Texture` and `PINCache` generates lots of warnings 😢 Maybe we should move them to new demo project

**Comment by gungorbasa** [comment]
thank you @onmyway133 :)

**Comment by gungorbasa** [comment]
Unfortunately all that warnings are inevitable if you are using `Texture`. However, its an amazing library. It just makes everything so smooth :D 
If you want, I can create a separate demo project for `Texture`. @onmyway133

**Comment by onmyway133** [comment]
@gungorbasa yeah, that would be great ❤️

**Comment by gungorbasa** [comment]
@onmyway133 ok. I will create another one in couple days and create a new merge request 😃

**Pull request merged**